readme.md

```diff
@@ -112,6 +112,20 @@ Repository
 
 Various implementors make use of the Javascript implementation [erdtman/cose-js](https://github.com/erdtman/cose-js) (forked [here](https://github.com/ehn-dcc-development/cose-js)) and the Java implementation [cose-wg/COSE-JAVA](https://github.com/cose-wg/COSE-JAVA) (forked [here](https://github.com/ehn-dcc-development/COSE-JAVA))
 
-# TODO
+## Overview of Member State Trust lists
 
-* Add Trustlist overview (document linking to trustlists from various states who publish)
+### Production
+
+Country | URL
+-------	| ---
+DE 		| https://de.dscg.ubirch.com/trustList/DSC/
+NL 		| https://webtooling.rdobeheer.nl/unpacker/?shit=https://verifier-api.coronacheck.nl/v9/verifier/public_keys
```

Comment:
Note: keep away from fan! ;)

Reply:
Weird, it's aligned in my editor :grumble: